website/docs/tutorials/react-patterns.md

```diff
@@ -58,14 +58,17 @@ export default function ImageWithCaption() {
 }
 ```
 
-#### Providing a context for a message
-The same text elements with different contexts are extracted with different IDs. For example, if the word "right" uses the following two definitions in two different locations, the word is translated differently and merged back into the application as different translation entries.
+#### Providing context for a message
 
-- correct as in "you are right"
+When using generated IDs, the same text elements with different contexts are extracted with different IDs. For example, if the word "right" uses the following two definitions in two different locations, the word is translated differently and merged back into the application as different translation entry.
+
+- correctness as in "you are right"
 - direction as in "turn right"
 
 If the same text elements meet the following conditions, the text elements are extracted only once and use the same ID.
 
+TODO conditions?
```

Comment:
in the current docs, it's not clear what those conditions are

Reply:
conditions is a "context" + "source string"

Example: 

"right" + "correctness" = <idHash1>
"right" + "direction" = <idHash2>

In the whole codebase whenever these pairs would be met, the same hash id would be generated.